### DISCUSSION TITLE
Let's create an Awesome list for JSON Schema!

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#12] [General]
Awesome lists are awesome: https://github.com/sindresorhus/awesome

We should create one and get a bit of buzz going around JSON Schema.

@kinlane Do you think such a list could interact (Could be in either direction or both) with the Postman Collections you've been building?
Maybe building such a listing from that would be a neat idea, but the data would need some flag to indicate it's suffeciently awesome to be in the listing. It's supposed to be a hand curated list, not just an everyting listing.

**Comment by kinlane** [comment]
Yes, I think gathering the following elements (which I am organizing via [public workspace](https://www.postman.com/postman/workspace/postman-open-technologies-json-schema/overview) via any means possible, especially discussions here is valuable:

- **Adoption** - Examples of JSON Schema adoption by tooling and service providers.
- **Articles** - Useful articles about JSON Schema (working to keep within last 2 years).
- **Books** - Any books, or chapters in books that are dedicated to JSON Schema.
- **Education** - Any training, workshops, or curriculum built around JSON Schema.
- **Government** - References to government agencies using JSON Schema.
- **People** - Who are the people contributing to and moving forward everything JSON Schema.
- **Tooling** - What are the tools build around the JSON Schema specification.

I am aggregating all of these things [here in a public workspace as APIs](https://www.postman.com/postman/workspace/postman-open-technologies-json-schema/overview), but happy to gather from posts here, via Twitter, and on Slack like we have been doing via the "in-the-wild" channel!!

**Comment by jviotti** [comment]
I'll self assign this to me. I'll start the "awesome" list repo + some automation as a personal repo and then we can evaluate and see if we move it to the org

**Reply by kinlane** [reply]
And look -- someone name kinlane already published the workspace!! ;-)  Thanks @jviotti !!

**Comment by jviotti** [comment]
Here is the work in progress: https://github.com/jviotti/awesome-jsonschema

**Reply by jdesrosiers** [reply]
The "Getting Started" section is going to need some vetting. I did a spot check of a couple of the links. The first was describing draft-03! :grimacing:. The second had significant errors. I can take a closer look next week.

The "Vocabularies" links don't point to anything. Maybe they should point the spec?

We already track tools and libraries on the main site. Duplicating that here is probably counterproductive.

**Reply by karenetheridge** [reply]
The list shouldn't be duplicated, but it should be linked to. What if someone finds this list and thinks that that's all there is?

**Reply by jviotti** [reply]
@jdesrosiers 

> The "Getting Started" section is going to need some vetting. I did a spot check of a couple of the links. The first was describing draft-03! 😬. The second had significant errors. I can take a closer look next week.

Sounds good! I took the existing public Postman workspace collection as a starting point, but didn't spend much time revising them. I'll add you as a contributor to the repo so you can also merge as you please. Ideally we can move this repo to the JSON Schema org afterwards.

**Reply by jviotti** [reply]
I added everyone from this group to the repo.

**Comment by karenetheridge** [comment]
This should also link to the official "implementations" page on the website, in case anyone coming across the Awesome list things that's that all there is (as opposed to a subset).
